numba/cuda/cudadrv/driver.py

```diff
@@ -1746,8 +1746,9 @@ def get_devptr_for_active_ctx(ptr):
     pointer.
     """
     devptr = c_void_p(0)
-    attr = enums.CU_POINTER_ATTRIBUTE_DEVICE_POINTER
-    driver.cuPointerGetAttribute(byref(devptr), attr, ptr)
+    if ptr != 0:
```

Comment:
the `ptr == 0` condition is never exercised by the test.  Can you add a test that ingest a zero-sized array from the cuda array interface?

Reply:
OK I'm on it.

Reply:
@sklam Done, see 3b700f7.

Reply:
wait, I was wrong. hold on.

Reply:
OK @sklam I got it right in d266b6f.